packages/manager/apps/dedicated/client/app/dedicated/server/consumption/consumption.controller.js

```diff
@@ -134,24 +136,9 @@ class ServerConsumptionCtrl {
     );
   }
 
-  // function quotaStatus () {
-  //   $scope.state.throttled = $scope.consumptionData.throttled.status;
-  //   if ($scope.state.overQuota) {
-  //     Alerter.alertFromSWS([
-  //       $translate.instant("server_consumption_warning_overquota", {
-  //         t0: $scope.consumptionData.throttled.speed
-  //       }),
-  //       //$translate.instant("server_consumption_warning_overquota_increase")
-  //     ].join(" "), null, $scope.alert);
-  //   } else if ($scope.state.nearQuota) {
-  //     Alerter.alertFromSWS([
-  //       $translate.instant("server_consumption_warning_nearquota", {
-  //         t0: $scope.consumption.total.text
-  //       }),
-  //       //$translate.instant("server_consumption_warning_nearquota_increase")
-  //     ].join(" "), null, $scope.alert);
-  //   }
-  // }
```

Comment:
Can you please remove this if not needed.

Reply:
yes `function quotaStatus` is already being removed